### DISCUSSION TITLE
Removing the space notation system

### DISCUSSION THREAD

**Discussion by rawsrc** [rawsrc/PhpEcho#19] [Polls]
Hi,

PhpEcho has a feature called "space notation". When it's active, if you define a value having a key containing a space, the space is transformed into a sub array.
```php
$block = new PhpEcho();
$block['abc def'] = true;
// becomes automatically
$block['abc']['def'] = true;
```
Personally, i used it once and my opinion is to remove it.
The complexity is just useless and also for debugging...

**Comment by laurentsc** [comment]
If some persons used it (maybe me, I've forgotten), after an update, maybe a risk of no compatibility of the code, no?
